### DISCUSSION TITLE
Can't index on Asustor NAS

### DISCUSSION THREAD

**Discussion by Wingstyle** [photoprism/photoprism#2232] [Q&A] [answered]
I have installed PhotoPrism on my Asustor AS5202T NAS and can't get it to Index any of my photo's. After I do a rescan in Photoprism Library, I get the following in the log:

2022-04-09 14:50:47 INFO indexing completed in 0 s
2022-04-09 14:50:47 INFO index: found no orphan files [133.056µs]
2022-04-09 14:50:47 INFO index: found no new or modified files
2022-04-09 14:50:47 INFO index: found no .ppignore file
2022-04-09 14:50:47 INFO indexing originals...

I can't find anyplace in PhotoPrism to change where it looks for photo's. Searching the internet leads me to believe I have to change something in Docker. I installed Portainer to see if I can figure out where to make this change. But being a complete novice at this I can't figure it out. I posted on the Asustor forum, but no help provided. I don't want to screw it up so any help someone can provide would be greatly appreciated.

**Comment by Genygo** [comment, marked as answer]
Hi there,
did you use this version? 
https://www.asustor.com/de/app_central/app_detail?id=1214&type=&model=50xx,51xx
Its looks like a community-maintained version of PP and a Farley old one too December 2020.
You could try to reach out to the maintainer.
Otherwise, the Asustor NAS have Docker already installed. You could use Portainer to install the official PP Container.
Maybe you find more information’s here:
https://www.asustor.com/en-gb/solution/what_is_docker

**Reply by Wingstyle** [reply]
Thank you Genygo.
That is the version I installed from Asustor App Central on my NAS. I tried to reach out to the maintainer but got a undeliverable message back. I also have a ticket going with Asustor technical for another issue, so I asked him if he would check into updating PP on App Central. If that doesn't work I may try to install PP with Portainer. Not knowing what I am doing makes that a second option :-)
